templates/pages/login_error.html.twig

```diff
@@ -53,6 +53,8 @@
                 <i class="ti ti-login"></i>
                 <span>{{ __('Log in again') }}</span>
             </a>
+
+            {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::DISPLAY_LOGIN')) }}
```

Comment:
Maybe a new hook `DISPLAY_LOGIN_ERROR` should be created to differentiate between the login page and the login error page 

If I take the example of the news plugin (which lets you add information to the home page), this change will also display the information on the error page